components/feature/addons/src/main/java/mozilla/components/feature/addons/amo/AddonCollectionProvider.kt

```diff
@@ -370,7 +418,14 @@ internal fun JSONObject.getSafeJSONArray(key: String): JSONArray {
         getJSONArray(key)
     }
 }
-
+internal fun JSONObject.getSafeTranslations(valueKey: String, language: String?): Map<String, String> {
+    return if (get(valueKey) is String) {
```

Comment:
nit: Here it would be good to comment why i.e. the response without a language specified contains an array of languages, the response with a language specified contains the string directly.